src/pages/lists/hooks/useFilters/useFilters.ts

```diff
@@ -16,16 +16,32 @@ export const useFilters = (filterConfig: filterConfigType) => {
 
   const filterCount = activeFilters?.length;
 
+  const saveFilters = (theFilters: SetStateAction<filterConfigType>) => {
+    setFilters(theFilters);
+    writeStorage(APPLIED_FILTERS_KEY, theFilters);
+  };
+
   const onChangeFilter = (e: any) => {
     const aFilters = { ...appliedFilters };
     aFilters[e.target.name] = e.target.checked;
-    setFilters(aFilters);
-    writeStorage(APPLIED_FILTERS_KEY, aFilters);
+    saveFilters(aFilters);
+  };
+
+  const onResetAll = () => {
+    saveFilters(filterConfig);
   };
 
-  const resetAll = () => {
-    setFilters(filterConfig);
+  const onClearGroup = (groupName: string) => {
+    const aFilters = { ...appliedFilters };
+
+    for (const name in aFilters) {
+      if (name.startsWith(groupName)) {
+        delete aFilters[name];
+      }
+    }
```

Comment:
This makes me wish there was something like `Array.prototype.filter()` for objects :(